Cargo.toml

```diff
@@ -108,12 +108,8 @@ zkevm-metrics = { path = "crates/metrics" }
 benchmark-runner = { path = "crates/benchmark-runner" }
 guest-libs = { path = "ere-guests/libs" }
 
-zkvm-interface = { git = "https://github.com/eth-applied-research-group/ere", tag = "v0.0.10", package = "zkvm-interface" }
-ere-sp1 = { git = "https://github.com/eth-applied-research-group/ere", tag = "v0.0.10", package = "ere-sp1" }
-ere-risc0 = { git = "https://github.com/eth-applied-research-group/ere", tag = "v0.0.10", package = "ere-risc0" }
-ere-pico = { git = "https://github.com/eth-applied-research-group/ere", tag = "v0.0.10", package = "ere-pico" }
-ere-openvm = { git = "https://github.com/eth-applied-research-group/ere", tag = "v0.0.10", package = "ere-openvm" }
-ere-zisk = { git = "https://github.com/eth-applied-research-group/ere", tag = "v0.0.10", package = "ere-zisk" }
+zkvm-interface = { git = "https://github.com/eth-applied-research-group/ere", rev = "63f4ab1fff482a9437ebea13c395d6a4a6f9d255", package = "zkvm-interface" }
```

Comment:
@han0110, do you feel now might be a good moment to cut a release? Or is it worth waiting a bit more if you're testing something extra?

Reply:
More thinking out loud...
I think until we have the v3 in Risc0, I won't be using it for benchmarks yet.

Would Zisk be usable for running benchmarks in single gpus as is?